packages/jest-runner/errorUtils.test.ts

```diff
@@ -31,31 +35,72 @@ describe("ErrorUtils", () => {
     at fulfilled (/home/norbert/Code-Intelligence/jazzer.js/packages/jest-runner/dist/fuzz.js:58:24)
 `;
 
-	it("clean up jest runner frames", () => {
-		expect(cleanupJestRunner(stack)).toEqual(`Error:
+	beforeEach(() => {
+		error.stack = stack;
+	});
+
+	describe("clean up Jest runner frames", () => {
+		it("in errors", () => {
+			expect(cleanupJestError(undefined)).toBeUndefined();
+			const cleanedUp = cleanupJestError(error);
+			expect((cleanedUp || {}).stack).toEqual(`Error:
```

Comment:
Not quite sure about this one. I'd have expected a `.toContain` or `toMatch` call here when testing for the **in** property. Am I missing something?

Reply:
True, changed.